test/GovernorCountingFractional.t.sol

```diff
@@ -665,23 +676,17 @@ contract GovernorCountingFractionalTest is Test {
     _voteSplit = _randomVoteSplit(_voteSplit);
 
     uint128 _forVotes = uint128(_weight.mulWadDown(_voteSplit.percentFor));
-    uint128 _againstVotes = uint128(_weight.mulWadDown(_voteSplit.percentAgainst));
+    uint128 _abstainVotes = uint128(_weight.mulWadDown(_voteSplit.percentAbstain));
 
-    uint32 _expectedState;
-    if (_forVotes >= _quorum && _forVotes > _againstVotes) {
-      _expectedState = uint32(IGovernor.ProposalState.Succeeded);
-    } else {
-      _expectedState = uint32(IGovernor.ProposalState.Defeated);
-    }
-
-    _quorumTest(_voterAddr, _weight, _voteSplit, _expectedState);
+    bool _wasQuorumReached = _forVotes + _abstainVotes >= _quorum;
+    _quorumTest(_voterAddr, _weight, _voteSplit, _wasQuorumReached);
   }
 
   function _quorumTest(
     address _voterAddr,
     uint256 _weight,
     FractionalVoteSplit memory _voteSplit,
-    uint32 _expectedState
+    bool _expectedQuorumReached
```

Comment:
Nit: much better name but I'd be even more explicit, something like `_isQuroumReachedExpected`